src/environmentd/tests/util.rs

```diff
@@ -252,6 +252,8 @@ pub fn start_server(config: Config) -> Result<Server, anyhow::Error> {
         segment_api_key: None,
         egress_ips: vec![],
         aws_account_id: None,
+        launchdarkly_sdk_key: None,
+        config_sync_loop_interval: Duration::ZERO,
```

Comment:
> The ZERO here won't cause a busy loop because the sdk key is None, and so doesn't ever start the ticker?

Yes, look for the following comments in the diff for the places in `environmentd::serve` that assert this:
```rust
// Initialize the system parameter frontend if `launchdarkly_sdk_key` is set.
// ...

// If system_parameter_frontend is present, start the system_parameter_sync loop.
// ...
```